pubspec.yaml

```diff
@@ -20,12 +24,19 @@ dependencies:
   hydrated_bloc: ^9.1.5
   intl: ^0.19.0
   json_annotation: ^4.9.0
+  lottie: ^3.1.2
+  nested_scroll_views: ^0.0.11
+  particles_flutter: ^0.1.4
```

Comment:
Why are these packages needed?

Reply:
![Screenshot 2024-08-28 at 2 09 15 PM](https://github.com/user-attachments/assets/29cfc170-107e-41a4-a959-4569eee3dc24)

We are currently using the `particles_flutter` package to display the particle effects in the background. The `nested_scroll_views` package is required for the puzzle screen. The quiz questions are rendered using the designated package, and we need the quiz content card to be scrollable while ensuring that the entire screen remains scrollable as well.

Reply:
I think we can achieve nested scroll views without a package and I’m not sure that the particles are really worth adding a whole new package.